kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```diff
@@ -135,6 +135,30 @@ spec:
                                 type: string
                                 minLength: 1
                                 maxLength: 253
+                          trustAnchorRef:
+                            description: |
+                              A reference to a resource holding the trusted CA certificates for validation of
+                              a Kafka client's TLS client certificate.
+                              If not specified the proxy will use the trust store inherited from the platform.
```

Comment:
"inherited from the platform" sounds fancy, but I don't know what it means. Does it mean inherited from the OS, or from the JRE? Someone might even reasonably misinterpret it to mean it's inherited from Kubernetes. We should be more specific.